data/brands/amenity/fuel.json

```diff
@@ -822,18 +808,6 @@
         "name": "Caltex"
       }
     },
-    {
-      "displayName": "CAMPSA",
```

Comment:
From taginfo, I noticed that there are a lot of `brand:1992=Campsa` and almost every that fuel station have `brand:wikidata=Q174747`, Is "Repsol" replaced all "CAMPSA"? If so, add "CAMPSA" as a matchName for "Repsol"

Reply:
Good catch, I have added matchName under "Repsol".